docs/open-source-tizen/tizen-rt/rt-getting-started.md

```diff
@@ -1,38 +1,77 @@
+2. Untar the `gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar` file and export to the following path:  
+`export PATH=/home/xyz/currentwork/workcommon/toolchain/armgcc492/bin:$PATH`
 
-In Ubuntu 13.10 and higher, if you cannot execute 32-bit packages, use the following command:
+    > **Note**  
+	> The above command is an example only - do not copy-paste it directly. If you have multiple toolchains, modify the path appropriately.
 
-`sudo apt-get install -y lib32z1 lib32ncurses5 lib32bz2-1.0`
+    In Ubuntu 13.10 and higher, if you cannot execute 32-bit packages, use the following command:
 
-## Getting Source Codes from Git
+    `sudo apt-get install -y lib32z1 lib32ncurses5 lib32bz2-1.0`
 
-Run the following commands:
+## Getting Source Code from Git<a name="git"></a>
 
+To get the source code, run the following commands:
 
+```bash
+    $ git clone ssh://<Your ID>@review.tizen.org:29418/rtos/tinyara
+    $ cd tinyara
+    $ TINYARA_BASEDIR="$PWD"
+    $ cd "$TINYARA_BASEDIR"
+```
 The `TINYARA_BASEDIR` environment variable is meant for reference and is used later.
 
-**※ To get the source codes from GitHub, use the following command instead of **`git clone ssh://...`**:**
+> **Note**  
+> To get the source code from GitHub, instead of `git clone ssh://...`, use the following command:  
+> ```bash
+> $ git clone https://github.com/Samsung/TinyAra.git
+> ```
 
-`$ git clone https://github.com/Samsung/TinyAra.git`
+## Building Tizen RT<a name="build"></a>
+To build Tizen RT:
 
-## Building Tizen RT
+1. Configure the build from the `$TINYARA_BASEDIR/os/tools` directory:
 
-1. Configure the build from the `$TINYARA_BASEDIR/os/tools` directory:`$ ./configure.sh <board>/<configuration_set>`For list of supported boards and configuration sets, see [Appendix A](https://source.tizen.org/documentation/tizen-rt/tizen-rt-getting-started#Appendix.A).For example: `./configure.sh sidk_s5jt200/hello_with_tash`The command copies the configuration set for the particular board into the `$TINYARA_BASEDIR/os` directory. The configuration can be modified through the `make menuconfig` command from the `$TINYARA_BASEDIR/os` directory.
-2. Initiate the build from the `$TINYARA_BASEDIR/os` directory:`$ make`The built binaries are located in the `$TINYARA_BASEDIR/build/output/bin` directory.
+   `$ ./configure.sh <board>/<configuration_set>`
 
-## Appendix A
+    For example:  
 
-**Board**
+   `./configure.sh sidk_s5jt200/hello_with_tash`
 
+    The command copies the configuration set for the particular board into the `$TINYARA_BASEDIR/os` directory. The configuration can be modified by running the `make menuconfig` command from the `$TINYARA_BASEDIR/os` directory.
+1. Initiate the build from the `$TINYARA_BASEDIR/os` directory:
 
-**Configuration Sets**
+   `$ make`
 
+    The built binaries are located in the `$TINYARA_BASEDIR/build/output/bin` directory.
+
+The following boards and configuration sets are supported:
```

Comment:
The remaining content in this topic has been added from TD, since the original topic contained headings only. Is this OK?